aspnet/mvc/views/partial/sample/src/PartialViewsSample/Views/Shared/Error.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@{
```

Comment:
it's just part of the template project (used with app.UseExceptionHandler("/Home/Error"); in startup)